authors/apps/articles/views.py

```diff
@@ -88,3 +89,123 @@ def delete(self, request, slug):
         return Response({
             'errors': 'sorry article with that slug doesnot exist'
         }, status=status.HTTP_404_NOT_FOUND)
+
+
+class LikeArticleAPIView(APIView):
```

Comment:
I think you might need to rethink the approach you've taken to implement this particular View. Can we sync so that we can get more clarity on how this was implemented? @Karuhanga